Fatal Python error: init_import_site: Failed to import the site module
Python runtime state: initialized
Traceback (most recent call last):
  File "/home/sek519/miniconda/lib/python3.9/site.py", line 73, in <module>
    import os
  File "/home/sek519/miniconda/lib/python3.9/os.py", line 27, in <module>
    import stat as st
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 982, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 925, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1423, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1395, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1526, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1569, in _fill_cache
OSError: [Errno 40] Too many levels of symbolic links: '/home/sek519/warm_starting/vws/experiments'
